CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ You can find its changes [documented below](#060---2020-06-01).
 - GTK: Don't crash when receiving an external command while a file dialog is visible. ([#1043] by [@jneem])
 - Fix derive `Data` when type param bounds are defined ([#1058] by [@chris-zen])
 - Ensure that `update` is called after all commands. ([#1062] by [@jneem])
+- Fix comparison between Modifier with NUM_LOCK and RawMods. ([#1077] by [@HoNile])
```

Comment:
The PR link should be added to the others, below the profile links.

Reply:
Done !